### PR TITLE
Add Router to MathsApp

### PR DESCRIPTION
**Pull request by Ibnballo1**
I completed the Math Magician website by doing the following:

- I created two other pages, the home page, and the quote page.
- I then use a Router to link up the pages together to make it SPA

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,8 +5,13 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable react/self-closing-comp */
 /* eslint-disable linebreak-style */
```
Comment (omarsalem7) [inline src/App.js]: - [ ] Kindly remove comment lines for linter errors and fix them manually or by running this command `npx eslint . --fix`

File: src/components/Calculator.js [hunk 2]
```diff
@@ -8,7 +8,6 @@
 /* eslint-disable react/prefer-stateless-function */
```
Comment (omarsalem7) [inline src/components/Calculator.js]: - [ ] Kindly remove these comments also as stated above.

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable linebreak-style */
```
Comment (omarsalem7) [inline src/components/Header.js]: - [ ] From here also as stated above

File: src/components/Home.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable linebreak-style */
```
Comment (omarsalem7) [inline src/components/Home.js]: - [ ] From here also as stated above

File: src/components/Quote.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable linebreak-style */
```
Comment (omarsalem7) [inline src/components/Quote.js]: - [ ] And here also as stated above

File: src/index.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable linebreak-style */
```
Comment (omarsalem7) [inline src/index.js]: - [ ] From here also as stated above

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Ibnballo1 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- Routes work as expected ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @Ibnballo1  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implimenting the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**